http/etag.ts

```diff
@@ -116,6 +116,38 @@ async function calcFileInfo(
  * @param options Various additional options.
  * @returns The calculated ETag.
  */
+export async function eTag(
+  entity: string | Uint8Array,
+  options?: ETagOptions,
+): Promise<string>;
+/**
+ * Calculate an ETag for an entity. When the entity is a specific set of data
+ * it will be fingerprinted as a "strong" tag, otherwise if it is just file
+ * information, it will be calculated as a weak tag.
```

Comment:
We should update the descriptions to orient themselves to their respective overloads.